<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Я не знаю как назвать окно</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Название сорта:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Степень обжарки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание вкуса:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена в рублях:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Объём в мл:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sortName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="roast">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="type">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Молотый</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В зёрнах</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="price">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>128000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="size">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>128000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Done">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>280</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
